test/unittests/lock/test_lock.py

```diff
@@ -25,37 +26,43 @@
 
 class TestLock(unittest.TestCase):
     def setUp(self):
-        if exists('/tmp/mycroft'):
-            rmtree('/tmp/mycroft')
+        if exists(os.path.join(tempfile.gettempdir(), "mycroft")):
+            rmtree(os.path.join(tempfile.gettempdir(), "mycroft"))
```

Comment:
use `'` like the rest of the file to keep it consistent. (here and other places in this file)